examples/14_react-tweet/vite.config.ts

```diff
@@ -2,10 +2,13 @@
 export default ({ mode }: { mode: string }) => ({
   ...(mode === 'development' && {
```

Comment:
I think this pattern might be confusing for some users :)

Reply:
so maybe we can remove it?

Reply:
hmm, but I want to emphasize and confirm that this is dev-only workaround. any other idea?

Reply:
a comment maybe? // This is a dev-only solution.

Reply:
Actually, we shouldn't externalize for production. Maybe we should use `if` statement to be more readable?
Like https://vitejs.dev/config/#conditional-config

Reply:
Yes, that can be good too, with a comment.

Reply:
send a pr when you have time.